libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5537,4 +5537,37 @@ void GCS_MAVLINK::send_high_latency() const
         0, // Field for custom payload.
         0); // Field for custom payload.
 }
-#endif // HAL_HIGH_LATENCY2_ENABLED
\ No newline at end of file
+#endif // HAL_HIGH_LATENCY2_ENABLED
+
+void GCS_MAVLINK::send_position_target_global_int()
+{
+    Location target;
+    if (!get_position_target_location(target)) {
```

Comment:
we need to ensure that the target location is in global frame, see in copter, we could be in AMSL

Reply:
calling     
```
// convert altitude frame to AMSL (this may use the terrain database)
    if (!target.change_alt_frame(Location::AltFrame::ABSOLUTE)) {
        return;
    }
```
should be cheap enough in all case here